Bulldozer.CSV/SampleCSVs/06 - Contribution.csv

```diff
@@ -1,4 +1,4 @@
-IndividualID,FundName,SubFundName,FundGLAccount,SubFundGLAccount,FundIsActive,SubFundIsActive,ReceivedDate,CheckNumber,Memo,ContributionTypeName,Amount,StatedValue,ContributionID,ContributionBatchID,ContributionCreditCardType,IsAnonymous,Gateway,ScheduledTransactionForeignKey,FundId,^Projects^Project^V
+IndividualID,FundName,SubFundName,FundGLAccount,SubFundGLAccount,FundIsActive,SubFundIsActive,ReceivedDate,CheckNumber,Memo,ContributionTypeName,Amount,StatedValue,ContributionID,ContributionBatchID,ContributionCreditCardType,IsAnonymous,Gateway,ScheduledTransactionForeignKey,FundId,SourceType,^Projects^Project^V
 25570361,General Fund,,41000,,1,,2/7/2016,123,,Check,25,25,1,1,,,,,1,
```

Comment:
Be sure to add extra columns to sample data as well, otherwise column association could be off.

Reply:
@nateh777 fixed. Should be good to merge.